Basic-Car-Maintenance/Documentation.docc/Documentation.md

```diff
@@ -16,3 +16,76 @@ Icons have the following colors:
 * Blue #253AA7
 * Black #000000
 * Orange #F78616
+
+## Firestore Collections
+
+**_alerts_**  
+The alerts collection contains system level alerts that will be visable to all users
+
+| Fields    | Type    |
+| --------- | ------- |
+| actionTex | Text    |
```

Comment:
```suggestion
| actionText | Text    |
```

This was a typo on my end, my bad, please fix this

Reply:
For "Type" are you meaning the Firestore type, or the Swift types? I am guessing Firestore, but can you write that explicitly somewhere?